recipes/spec2nexus/meta.yaml

```diff
@@ -0,0 +1,66 @@
+{% set name = "spec2nexus" %}
+{% set version = "2021.2.0rc0" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 33fea814b07ce18c77d8b9390be54ef8a7a08f9d8ceb39afdd05ac99442a9f3d
+
+build:
+  noarch: python
+  number: 0
+  script: "{{ PYTHON }} -m pip install . -vv"
+
+requirements:
+  build:
+    - python >=3.7
+    - setuptools
+    - h5py
+    - lxml
+    - matplotlib
+    - numpy
+    - six
```

Comment:
Since this package is noarch. It shouldn't a build section. It should probably only need a host section with python, pip, and setuptools.

Reply:
@carterbox:  Thanks, I'll do that next